src/appkit.cs

```diff
@@ -11867,6 +11868,16 @@ partial interface NSResponder : NSCoding {
 		[Mac (10,12)]
 		[Export ("newWindowForTab:")]
 		void GetNewWindowForTab ([NullAllowed] NSObject sender);
+
+		[Export ("presentError:")]
+		bool PresentError (NSError error);
+
+		[Export ("willPresentError:")]
+		NSError WillPresentError (NSError error);
+
+		[EditorBrowsable (EditorBrowsableState.Advanced)] // You probably shouldn't override this method, because you have no way of reliably predicting whether this method vs. -presentError will be invoked for any particular error.
```

Comment:
AFAICS the docs says **not to override** but it seems to be useful to call it. That makes it a candidate for `[Sealed]` (to avoid it being overridden) and not a `[Advanced]` (which is a short form the the above).

However the native API is not a cute one has it exposes a selector (and not a C# delegate). FWIW I think it should be commented and the bug not closed until we provide a nicer, C#-py, API.